plane/source/docs/quadplane-auto-mode.rst

```diff
@@ -55,6 +55,15 @@ will be done at whatever height the aircraft is at when it starts on
 the NAV_VTOL_LAND waypoint. So you would typically want the previous
 waypoint to have an altitude of about 20 meters above the ground.
 
+If Q_OPTIONS is set for "Use a fixed wing approach" (bit 4), then instead of transitioning to VTOL flight and doing a VTOL landing, it will remain in plane mode , and proceed to the landing position,climbing or descending to the altitude set in the NAV_VTOL_LAND waypoint. When it reaches within Q_FW_LND_APR_RAD of the landing location, it will perform a LOITER_TO_ALT to finish the climb or decent to that altitude set in the waypoint, then, turning into the wind, transition to VTOL mode and proceed to the landing location and land.
+
+Note that if Q_FW_LND_APR_RAD=0, the Waypoint loiter radius will be used. Be sure to set the VTOL_LAND waypoint altitude and loiter radius to avoid obstacles. Note also, that if this waypoint is reached while in VTOL mode, it will proceed as a normal VTOL_LAND command, even if the option is set.
```

Comment:
You can't set the radius on the landing waypoint to the best of my knowledge. It only uses the vehicles loiter radius, or the `Q_FW_LND_APR_RAD` nothing on the waypoint itself.

Reply:
changed to Q_FW_LND_APR_RAD for clarity